gymnasium/vector/vector_env.py

```diff
@@ -52,31 +54,24 @@ class VectorEnv(Generic[ObsType, ActType, ArrayType]):
         In other words, a vector of multiple different environments is not supported.
     """
 
-    spec: "EnvSpec" = None
-
-    observation_space: gym.Space = None
-    action_space: gym.Space = None
+    metadata: dict[str, Any] = {}
+    render_mode: str | None = None
+    spec: EnvSpec | None = None
+    is_vector_env: int = True
```

Comment:
You mean `is_vector_env`, agreed. We should be able to remove this.
I think the that `Wrapper` supports vector environment shouldn't exist as well
Gymnasium has vector wrappers that we know support vectorisation otherwise users need to make their own so the variable is not needed